modular_citadel/code/game/machinery/cryopod.dm

```diff
@@ -138,48 +148,13 @@
 	var/despawn_world_time = null          // Used to keep track of the safe period.
 
 	var/obj/machinery/computer/cryopod/control_computer
+	var/item_storage_type = /obj/structure/closet/cabinet/cryo_drop			//with how storage components work this can be anything the player can open or anything with a storage component.
 	var/last_no_computer_message = 0
 
-	// These items are preserved when the process() despawn proc occurs.
-	var/list/preserve_items = list(
```

Comment:
These are considered round-important. that's why it's a specific list of preservatives as a last-ditch emergency check from people not putting their shit away.

Reminder that you can access the items with armory or bridge access.